ckan/config/middleware/flask_app.py

```diff
@@ -59,6 +60,8 @@
 from ckan.types import CKANApp, Config, Response
 
 log = logging.getLogger(__name__)
+# do not use _csrf unless you know what you're doing
+_csrf = CSRFProtect()
```

Comment:
Let's call it `csfr` since by design is not a private variable (we are accesing it from other modules).

Also, let's remove or change the comment. Is there a specific scenario in mind that can break everything? If so, it would be better to document it. If not I don't find the comment being useful.